channels/serializers_test.py

```diff
@@ -397,3 +397,15 @@ def test_reported_comment():
     reported_content.user_reports = [['spam', 1]]
     reported_content.mod_reports = [['spam', 'jane'], ['junk', 'jow']]
     assert ReportedContentSerializer().get_reasons(reported_content) == sorted(['spam', 'junk'])
+
+
+# def test_serialize_missing_author(reddit_submission_obj, reddit_comment_obj):
```

Comment:
@noisecapella Forgot about this test. This checks to ensure that a blank user results in `None` values for the  serialized object author fields for the ES documents. This is different from the normal serializer, which yields `'[deleted]'` if there is a blank user. I'm guessing we want to keep that difference?

Reply:
I feel like a serializer should return `None` for both cases and then show `[deleted]` in the react UI if applicable. But I'm fine keeping the difference since it's out of scope